src/components/measureList/MeasureList.tsx

```diff
@@ -272,6 +272,15 @@ export default function MeasureList(props: {
     }
   };
 
+  const getModelFamily = (model: string) => {
```

Comment:
is there any way we could add a test case for this?

Reply:
I just added test cases for them